Resources/Locale/ru-RU/_white/prototypes/catalog/fills/crates/npc.yml

```
@@ -0,0 +1,2 @@
+ent-CrateNPCGondola = ящик с Гондолой
+    .desc = Ящик содержащий в себе одну Гондолу.
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)